charts/kube-state-metrics/values.yaml

```diff
@@ -37,8 +37,11 @@ autosharding:
 
 replicas: 1
 
-# Change the deployment strategy when autosharding is disabled
-# updateStrategy: Recreate
+# Change the deployment strategy when autosharding is disabled.
+# The default is "Recreate" in favor of preventing two instances from running, but this might create a small gap in data.
+# When using "RollingUpdate", it's wise to use a pessimistic approach when querying for "kube-state-metrics" and perform something like this:
+# max by(namespace, pod) (kube_pod_container_status_restarts).
+# updateStrategy: RollingUpdate
```

Comment:
I would rework this paragraph taking into account that `RollingUpdate` is the default.
You can also add : 
```yaml
# ref: https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy
```